### PR TITLE
RVAT-573_add test for check existence sign up form

### PR DESCRIPTION
**Pull request by JugerJuger**

### CODE REVIEW

File: src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java [hunk 1]
```diff
@@ -1,7 +1,56 @@
...
-	
+	private WebDriver driver;
+	private WebDriverWait wait;
+
+	@BeforeClass
+	public void beforeClass() {
+		ChromeDriverManager.getInstance().setup();
+		driver = new ChromeDriver();
+		driver.get("https://green-tourism.herokuapp.com");
+		wait = new WebDriverWait(driver, 10);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java]: Move it the string const MAX_WAIT_TIME

File: src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+						"html.ng-scope body header nav.navbar.navbar-default.navbar-fixed-top.navbar-style.ng-scope div.container.nav-padding div#navbar.collapse.navbar-collapse ul.nav.navbar-nav li.dropdown.open ul.dropdown-menu.dropdown-auth.nav-ul auth.ng-isolate-scope div.ng-scope div.container.registration.ng-scope form.ng-pristine.ng-invalid.ng-invalid-required.ng-valid-minlength.ng-valid-maxlength.ng-valid-email.ng-valid-pattern.ng-valid-match")),
+				"Signup form is not exist!!!");
+
+	}
+
+	protected boolean isExist(By element) {
+		try {
+			driver.findElement(element);
+			return true;
+		} catch (NoSuchElementException e) {
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java:52]: What if you have received another exception?

File: src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java (hunk 1 above)
Comment (JugerJuger) [reply src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java]: ok

File: src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemFireFox.java [hunk 3]
```diff
@@ -1,7 +1,57 @@
...
+						"html.ng-scope body header nav.navbar.navbar-default.navbar-fixed-top.navbar-style.ng-scope div.container.nav-padding div#navbar.collapse.navbar-collapse ul.nav.navbar-nav li.dropdown.open ul.dropdown-menu.dropdown-auth.nav-ul auth.ng-isolate-scope div.ng-scope div.container.registration.ng-scope form.ng-pristine.ng-invalid.ng-invalid-required.ng-valid-minlength.ng-valid-maxlength.ng-valid-email.ng-valid-pattern.ng-valid-match")),
+				"Signup form is not exist!!!");
+
+	}
+
+	protected boolean isExist(By element) {
+		try {
+			driver.findElement(element);
+			return true;
+		} catch (NoSuchElementException e) {
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemFireFox.java:52]: What if you have received another exception?

File: src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java (hunk 2 above)
Comment (JugerJuger) [reply src/test/java/rv/atqc/gttesting/possibility_sign_up_into_system/PossibilitySignUpIntoSystemChrome.java:52]: test will be failed but not in object of testing

**Review by dmatsev — APPROVED** [review]

**Review by TyshchenkoRoma — APPROVED** [review]

**Review by MaxKovalets — APPROVED** [review]

**Pull request merged**